cli/cmd/runtime/start.go

```diff
@@ -68,6 +68,8 @@ type Config struct {
 	ActivitySinkKafkaBrokers string `default:"" split_words:"true"`
 	// Kafka topic of an activity client's sink
 	ActivitySinkKafkaTopic string `default:"" split_words:"true"`
+	GlobalDrivers          string `split_words:"true"`
+	PrivateDrivers         string `split_words:"true"`
```

Comment:
1. On the runtime-level, should there not only be one map? And then a separate one can be provided on the instance-level (when calling `CreateInstance`)

Reply:
I thought since this is going to be fixed/mandatory for every instance we can keep them as part of runtime config itself to group them. 
I can add them as part of instance as well.